generate.ts

```diff
@@ -18,8 +23,17 @@ interface SdkGlobals {
     apiSrcDescription: string;
     apiCache: { [key: string]: any; }
     sdkDocsByMethodName: { [key: string]: SdkDoc; }
+    specialization: string;
 }
 
+const defaultApiSpecFilePath = "../API_Specs"; // Relative path to Generate.js
+const defaultApiSpecGitHubUrl = "https://raw.githubusercontent.com/PlayFab/API_Specs/master";
+const defaultApiSpecPlayFabUrl = "https://www.playfabapi.com/apispec";
+const tocFilename = "TOC.json";
+const tocCacheKey = "TOC";
+const specializationTocCacheKey = "specializationTOC";
+const defaultSpecialization = "sdk";
```

Comment:
I was initially confused by this diff, because I thought these vars existed already. Then I scrolled and I saw you moved them. Did you move them up for readability?
I don't oppose the change, but I'd like to have some insight as to why you did it.

Reply:
I guess also, how do you feel about maybe loading those default paths from the TOC for me?  😄 😄 😄  (In a future checkin)

Reply:
Yep!

---
In reply to: [198675155](https://github.com/PlayFab/SDKGenerator/pull/144#discussion_r198675155) [](ancestors = 198675155)

Reply:
Moved them up a little so that we can use them in SdkGeneratorGlobals initialization below

---
In reply to: [198674114](https://github.com/PlayFab/SDKGenerator/pull/144#discussion_r198674114) [](ancestors = 198674114)